cryptoorgchain/chain.json

```diff
@@ -31,33 +31,33 @@
   },
   "codebase": {
     "git_repo": "https://github.com/crypto-org-chain/chain-main",
-    "recommended_version": "v4.2.2",
+    "recommended_version": "v4.2.6",
     "compatible_versions": [
-      "v4.2.2"
+      "v4.2.6"
     ],
     "binaries": {
-      "linux/amd64": "https://github.com/crypto-org-chain/chain-main/releases/download/v4.2.2/chain-main_4.2.2_Linux_x86_64.tar.gz",
-      "linux/arm64": "https://github.com/crypto-org-chain/chain-main/releases/download/v4.2.2/chain-main_4.2.2_Linux_arm64.tar.gz",
-      "darwin/amd64": "https://github.com/crypto-org-chain/chain-main/releases/download/v4.2.2/chain-main_4.2.2_Darwin_x86_64.tar.gz",
-      "darwin/arm64": "https://github.com/crypto-org-chain/chain-main/releases/download/v4.2.2/chain-main_4.2.2_Darwin_arm64.tar.gz",
-      "windows/amd64": "https://github.com/crypto-org-chain/chain-main/releases/download/v4.2.2/chain-main_4.2.2_Windows_x86_64.zip"
+      "linux/amd64": "https://github.com/crypto-org-chain/chain-main/releases/download/v4.2.6/chain-main_4.2.6_Linux_x86_64.tar.gz",
+      "linux/arm64": "https://github.com/crypto-org-chain/chain-main/releases/download/v4.2.6/chain-main_4.2.6_Linux_arm64.tar.gz",
+      "darwin/amd64": "https://github.com/crypto-org-chain/chain-main/releases/download/v4.2.6/chain-main_4.2.6_Darwin_x86_64.tar.gz",
+      "darwin/arm64": "https://github.com/crypto-org-chain/chain-main/releases/download/v4.2.6/chain-main_4.2.6_Darwin_arm64.tar.gz",
+      "windows/amd64": "https://github.com/crypto-org-chain/chain-main/releases/download/v4.2.6/chain-main_4.2.6_Windows_x86_64.zip"
     },
     "genesis": {
       "genesis_url": "https://github.com/crypto-org-chain/mainnet/raw/main/crypto-org-chain-mainnet-1/genesis.json"
     },
     "versions": [
       {
-        "name": "v4.2.2",
-        "recommended_version": "v4.2.2",
+        "name": "v4.2.6",
```

Comment:
Perhaps the name 'v4.2' would be more appropriate as it is generalized for both v4.2.2 and v4.2.6.

Reply:
@JeremyParish69 I think we need to take this offline, but yeah, changing name doesn't make sense.